plugintypes/commit.go

```diff
@@ -11,11 +11,12 @@ import (
 // ---[ Observation ]-----------------------------------------------------------
 
 type CommitPluginObservation struct {
-	NewMsgs     []cciptypes.RampMessageHeader   `json:"newMsgs"`
-	GasPrices   []cciptypes.GasPriceChain       `json:"gasPrices"`
-	TokenPrices []cciptypes.TokenPrice          `json:"tokenPrices"`
-	MaxSeqNums  []SeqNumChain                   `json:"maxSeqNums"`
-	FChain      map[cciptypes.ChainSelector]int `json:"fChain"`
+	NewMsgs     []cciptypes.RampMessageHeader `json:"newMsgs"`
+	GasPrices   []cciptypes.GasPriceChain     `json:"gasPrices"`
+	TokenPrices []cciptypes.TokenPrice        `json:"tokenPrices"`
+	MaxSeqNums  []SeqNumChain                 `json:"maxSeqNums"`
+	// TODO: Why is this a map and not a list?
+	FChain map[cciptypes.ChainSelector]int `json:"fChain"`
```

Comment:
The other types are related to commit report and seemed better to be a slice (e.g. for determinism and matching onchain types).

FChain is not part of the report, determinism not needed anywhere so seemed easier to keep as a map.